### PR TITLE
Fix Snap launch of OpenSCAD

### PR DESCRIPTION
**Pull request by chennes**
Patch provided by forums user 'lamikr': https://forum.freecadweb.org/viewtopic.php?f=10&t=40848

Note that I have not yet tested this patch, I am just submitting on behalf of its author.

### CODE REVIEW

File: src/Mod/OpenSCAD/OpenSCADUtils.py [hunk 1]
```diff
@@ -142,12 +142,31 @@ def getopenscadversion(osfilename=None):
...
+        try:
+            env.pop('LD_LIBRARY_PATH')
+        except KeyError:
+            None
+        def check_output2(*args, **kwargs):
+            env = os.environ.copy()
+            try:
+                env.pop('LD_LIBRARY_PATH')
+            except KeyError:
+                None
```
Comment (benj5378) [inline src/Mod/OpenSCAD/OpenSCADUtils.py:155]: Aren't line 151-155 duplicate of 146-150? The commands are already run once?

**Comment by luzpaz** [comment]
Is this Snap as in the Snap packaging ?

**Comment by adrianinsaval** [comment]
Also, is this something that we want for all distributions of FreeCAD? IMO this should be managed in the snap or at the very least be inside an if statement for the snap

**Comment by adrianinsaval** [comment]
Looking at the previous discussion this was actually a fix for the appimage, is this relevant for the snap too as stated in the title? IMO this should preferably be a configurable option that is patched to default to true for the appimage/snap/flatpak? but defaults to false normally.

**Comment by luzpaz** [comment]
Please advise

**Comment by chennes** [comment]
There is another ongoing forums discussion with a related, though more comprehensive, fix: https://forum.freecadweb.org/viewtopic.php?f=10&t=70790

**Comment by luzpaz** [comment]
Is there TL;DR summary somewhere ?

**Comment by luzpaz** [comment]
bump

**Comment by chennes** [comment]
No, this just fell off my radar. I don't think the code in this PR is quite right, it probably makes the most sense to just close it. We could open an issue if there isn't one already.

**Comment by luzpaz** [comment]
@chennes please advise what to name ticket to replace this PR ?

**Comment by chennes** [comment]
I guess just something like "Snap launch of OpenSCAD broken"

**Comment by adrianinsaval** [comment]
IIRC appimages had this problem too

**Pull request closed without merging**